qemu/tests/qemu_guest_agent.py

```diff
@@ -243,6 +246,35 @@ def gagent_check_install(self, test, params, env):
         pass
 
     @error_context.context_aware
+    def gagent_check_install_uninstall(self, test, params, env):
+        """
+        Repeate install/uninstall qemu-ga package in windows guest
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environmen.
+        """
+        repeats = int(params.get("repeat_times", 1))
+        error_context.context("Repeat install/uninstall qemu-ga-xx.msi for "
+                              "%s times" % repeats, logging.info)
+
+        if not self.vm:
+            vm = self.env.get_vm(params["main_vm"])
+            vm.verify_alive()
+            self.vm = vm
```

Comment:
self.vm = self.env.get_vm(params["main_vm"])
self.vm.verify_alive()

Reply:
yes, vm variable is redundant.
modified, pls help review. 
thanks